src/components/CohortBuilder/Summary/DemographicChart.js

```diff
@@ -80,15 +80,24 @@ export const demographicQuery = sqon => ({
   `,
   variables: { sqon },
   transform: data => {
-    const toChartData = ({ key, doc_count }) => ({
-      id: keyToId(key),
-      label: key,
-      value: doc_count,
-    });
+    const toChartData = ({ key, doc_count }) => {
+      const dataKey = keyToDisplay(key === DATA_MISSING ? 'No Data' : key);
+      return {
+        id: dataKey,
+        label: dataKey,
+        value: doc_count,
+      };
+    };
+
+    const DATA_MISSING = '__missing__';
```

Comment:
caught a little confusion about where `DATA_MISSING` was defined :P let's just move this above `toChartData`? :)